File: client/WorkoutLogger/web/com/jessewarden/workoutlogger/vo/WorkoutVO.dart
part of workoutloggerlib;


class WorkoutVO
{
  String name;
  DateTime occurrence;
  String type = WorkoutTypes.DEFAULT;
  int totalTimeInMilliseconds;
  int averageTimeInMilliseconds;
  List<CircuitVO> circuits;
  
  WorkoutVO()
  {
    
  }

}

File: client/WorkoutLoggerJS/src/com/jessewarden/workoutlogger/vo/WorkoutVO.js
define([], function()
{

	function WorkoutVO(name, occurrence, workoutType, totalTimeInMilliseconds, averageTimeInMilliseconds, exercises)
	{
		this.name       = name;
		this.occurrence  = occurrence;
		this.workoutType    = workoutType;
		this.totalTimeInMilliseconds = totalTimeInMilliseconds;
		this.averageTimeInMilliseconds = averageTimeInMilliseconds;
		this.exercises    = exercises;
	}

	return WorkoutVO;

});